contracts/CO30_Jonty_JackalsLoot.Woodland_ACR/initPlayerLocal.sqf

```diff
@@ -1,27 +1,27 @@
 #include "do_not_edit\script_component.hpp"
 #include "do_not_edit\initPlayerLocal.sqf"
-#include "speak.sqf"
 
 [
     // Add additional files here
     // if you create "edit_me/briefing/example.html, then add "Example" here
 ] call mission_fnc_briefing;
 
-mission_fnc_addIntel = {
-  player createDiaryRecord ["Diary", ["Missing Inventory", "- Add your jot notes<br/>- Here"]];
-  missionNamespace setVariable ["intelJip", 1, true];
-};
-[convoyOff, "Speak01"] call mission_fnc_speak;
-["mission_speak", {
-  params ["_unit"];
-  if (_unit != convoyOff) exitWith {};
-  if (missionNamespace getVariable ["intelJip", false]) exitWith {};
-  call mission_fnc_addIntel;
-}] call CBA_fnc_addPerFrameHandler;
-
-if (missionNamespace getVariable ["intelJip", false]) then {
-  call mission_fnc_addIntel;
-};
+["mission_intel", {
+    player createDiaryRecord ["Diary", ["Missing Inventory", "The convoy was hoping to avoid attention by taking these secluded roads to avoid attention but it made the perfect opportunity for Miyović's guys to raid their assets. These trucks were transporting vital military assets – missile and air-to-ground bombing units. Miyović might not be able to deploy these complex weapons systems himself, but we fear his forces could extract the deadly warheads and unleash catastrophic damage. We have to get those missiles secured before they fall into the wrong hands. CDF has been constantly carrying out UAV surveillance and has tracked high activty in the red zone marked on the map. That may be a good place to start your search."]];
```

Comment:
Now that you have the audio, this should be shortened or include the jot note format of what the players need to do